### DISCUSSION TITLE
Failed to compile: @smithy/util-retry/dist-es/AdaptiveRetryStrategy.js 10:17 Module parse failed: Unexpected token (10:17) File was processed with these loaders

### DISCUSSION THREAD

**Discussion by drewg2009** [aws/aws-sdk-js-v3#5783] [Q&A]
After installing the Javascript AWS SDK to a React App, I'm getting the following error when I startup the application:

Failed to Compile

```
./node_modules/@smithy/util-retry/dist-es/AdaptiveRetryStrategy.js 10:17
Module parse failed: Unexpected token (10:17)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|     const {
|       rateLimiter
>     } = options ?? {};
|     this.rateLimiter = rateLimiter ?? new DefaultRateLimiter();
|     this.standardRetryStrategy = new StandardRetryStrategy(maxAttemptsProvider);
```

I installed the AWS Secrets Manager Client following the following guide and running this command:

`yarn add @aws-sdk/client-secrets-manager `
[https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/installing-jssdk.html](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/installing-jssdk.html)

**Comment by julien-tamade** [comment]
Hey did you ever solve this? Thanks!
